samples/features/azure-arc-data-controller/deployment/kubeadm/ubuntu-single-node-vm/cleanup-controller.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run as root"
+  exit
+fi
+DIR_PREFIX=$1
+
+kubeadm reset --force
+
+# Clean up azdata-cli package.
+#
+unalias azdata
+sudo dpkg --remove --force-all azdata-cli
+
+systemctl stop kubelet
+rm -rf /var/lib/cni/
+rm -rf /var/lib/etcd/
+rm -rf /run/flannel/
+rm -rf /var/lib/kubelet/*
+rm -rf /etc/cni/
+rm -rf /etc/kubernetes/
+
+ip link set cni0 down
+#brctl delbr cni0
+ip link set flannel.1 down
+#brctl delbr flannel.1
+iptables -F && iptables -t nat -F && iptables -t mangle -F && iptables -X
+
+rm -rf .azdata/
+rm -rf bdcdeploy/
```

Comment:
Seems like this shouldn't be required since you are not doing pip install.

Reply:
Removed.